doc/source/install.rst

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
I didn't realize the main `README.rst` was generated by Sphinx based on some template until now. I guess to fix the issues I pointed out in `README.rst`, you have to tweak the template.

Reply:
rst and conf.py (except headers and comments) are written by me. html and pdf are generated by sphinx via make and conf.py for the web version of the doc.